app/common/commonMenu.js

```diff
@@ -28,6 +28,15 @@ const ensureAtLeastOneWindow = (frameOpts) => {
   }
 }
 
+const getCurrentWindowId = () => {
```

Comment:
actually in this case it's even simpler because you can use -2 to specify the current window (or leave it empty)